modules/introduction-bias-monitoring.adoc

```diff
@@ -0,0 +1,35 @@
+:_module-type: PROCEDURE
+
+[id="introduction-bias-monitoring_{context}"]
+== Introduction
+
+Ensuring that your machine learning models are fair and unbiased is essential for building trust with your users. Although fairness can be assessed during model training, it is only in deployment that your models encounter real-world data. Even if your models are unbiased on training data, they can exhibit dangerous biases in real-world scenarios. Therefore, it is crucial to monitor your models for fairness during their real-world deployment.
+
+
+In this tutorial, you learn how to monitor models for bias. You will use two example models to complete the following tasks:
+
+* Deploy the models by using `{productname-short}` model serving
+* Send training data to the models
+* Examine the metadata for the models
+* Check the fairness for the models
+* Schedule and check fairness and identity metric requests
+* Simulate real-world data
+
+
+== About the example models
+We will take on the persona of a dev-ops engineer for a credit lender. Our data scientists have created two candidate neural networks to predict if a borrower will default on the loan they hold with us. Both models use the following information about the applicant to make their prediction:
```

Comment:
For this tutorial, your role is the dev-ops engineer for a credit lending company. The company’s data scientists have created two candidate neural network models to predict whether a borrower will default on a loan. Both models make their predictions based on the following information from the borrower’s application: